lib/crate_ruby/client.rb

```diff
@@ -163,31 +166,29 @@ def blob_get(table, digest)
     def blob_delete(table, digest)
       uri = blob_path(table, digest)
       @logger.debug("BLOB DELETE #{uri}")
-      req = Net::HTTP::Delete.new(uri)
+      req = Net::HTTP::Delete.new(uri, headers)
       response = request(req)
-      success = case response.code
-                  when "200"
-                    true
-                  else
-                    @logger.info("Response #{response.code}: #{response.body}")
-                    false
-                end
-      success
+      case response.code
+      when '200'
+        true
+      else
+        @logger.info("Response #{response.code}: #{response.body}")
+        false
+      end
     end
 
-
     # Return the table structure
     # @param [String] table_name Table name to get structure
     # @param [ResultSet]
     def table_structure(table_name)
-      execute("select * from information_schema.columns where table_schema = 'doc' AND table_name = '#{table_name}'")
+      execute('select * from information_schema.columns where table_schema = ?' \
+              'AND table_name = ?', [schema, table_name])
     end
 
-
     def insert(table_name, attributes)
       vals = attributes.values
-      binds = vals.count.times.map {|i| "$#{i+1}"}.join(',')
-      stmt = %Q{INSERT INTO "#{table_name}" (#{attributes.keys.join(', ')}) VALUES(#{binds})}
+      binds = vals.count.times.map { |i| "$#{i + 1}" }.join(',')
+      stmt = %{INSERT INTO "#{table_name}" (#{attributes.keys.join(', ')}) VALUES(#{binds})}
```

Comment:
the column identifiers should also be double-quotes
the same way `#{table_name}` is

Reply:
@chaudum hm, can you post me an example how the bind values should look like, tried 

    INSERT INTO "posts" (id, title) VALUES("$1","$2")

which fails with

     Validation failed for id: Invalid value 'to_string(\"$1\")' in insert statement

Reply:
damn, just realized I'm working with the values instead of the identifiers